playbooks/keystone_voms.yaml

```diff
@@ -6,7 +6,7 @@
     # XXX This should be configurable
     - name: Add default VOs LSC files
       copy:
-        src: ./lsc/
+        src: ./templates/lsc/
```

Comment:
not to be a pedant, but "copy" takes static files. These should be taken from `files/` instead of `templates/`

I can't find where this is written in the documentation, so perhaps I am remembering incorrectly.

Reply:
Done.